config/locales/pt-BR.yml

```diff
@@ -23,6 +23,7 @@ pt-BR:
       updated_notice: Fórum de mensagem foi atualizado
     messageboard_group:
       create: Criar um novo grupo de mensagens
+      saved: Groupo de mensagens %{name} salvou
```

Comment:
Needs a fix, I'll do it today later.
Sorry for being away for awhile but was overloaded with some work.